cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,34 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
```

Comment:
> I think the best option is to remove the last commit and show both syntax errors and approval requirements as ACE editor annotations.

Not ideal, but an option if it is looking hard to use the normal `FormValidation` UI: script approval warnings are not common in practice, since if you are not an admin the sandbox mode is checked by default, so not many people would see this anyway.